### PR TITLE
add synbody_utils.py helps to use synbody dataset

### PR DESCRIPTION
**Pull request by Maoxie**
Add `io/exr_reader.py` & `utils/synbody_utils.py` helps to use synbody dataset.

Provide the following class:
- io/exr_reader.py
  - ExrReader
- utils/synbody_utils.py
  - SeqDataReader
  - SynbdoyExrReader
  - SynbodyReader

### CODE REVIEW

File: python/xrprimer/utils/synbody_utils.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+import flow_vis
+import Imath
+import OpenEXR
+import numpy as np
+
+PathLike = Union[str, Path]
+
+
+class ExrReader:
```
Comment (yl-1993) [inline python/xrprimer/utils/synbody_utils.py:21]: As offline discussion, we may extract the general part of `ExrReader` to `io`, and write a `SynBodyReader` to read all necessary data from synbody.

File: python/xrprimer/utils/synbody_utils.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    """Load 'seq_data.json' files, which contain sequences composition information.
+    """
+    def __init__(self, seq_data_path: PathLike) -> None:
+        with open(seq_data_path, 'r') as f:
+            seq_data = json.load(f)
+        self.seq_data: Dict = seq_data
+
+    def get_mask_colors(self):
+        """Load seq_data.json, get mask colors (rgb).
+        'seq_data.json'
```
Comment (yl-1993) [inline python/xrprimer/utils/synbody_utils.py:96]: Shall we also doc about the return value?

**Review by yl-1993 — APPROVED** [review]

File: python/xrprimer/utils/synbody_utils.py (hunk 1 above)
Comment (Maoxie) [reply python/xrprimer/utils/synbody_utils.py]: Done with `io/exr_reader.py`

File: python/xrprimer/utils/synbody_utils.py (hunk 2 above)
Comment (Maoxie) [reply python/xrprimer/utils/synbody_utils.py]: More detailed docstrings added.

**Pull request merged**